src/Microsoft.Extensions.PlatformAbstractions.Internal/project.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I don't think we should be adding a new assembly.  Instead can't this code all go inside the existing `Microsoft.DotNet.InternalAbstractions` project?

That way we don't have to ship a new NuGet package (which it doesn't look like you are publishing here), sign another assembly, etc.